vrs/vrs-annotation-batch-wrapper.py

```diff
@@ -0,0 +1,245 @@
+
+    # Checkpoint (write) resulting annotated table
+    ht_annotated = ht_annotated.checkpoint(
+        f"gs://{working_bucket}/vrs-temp/outputs/VRS-{prefix}", overwrite=args.overwrite
+    )
+    logger.info("Annotated Hail Table checkpointed")
+
+    if "test" not in args.version:
+        # NOTE: this construction is very slow on a large scale, maybe consider a Join ?
+        logger.info("Adding VRS ids to original Table")
+        ht_final = ht_original.annotate(
+            info=ht_original.info.annotate(
+                VRS_Allele=ht_annotated[
+                    ht_original.locus, ht_original.alleles
+                ].VRS_Allele
+            )
+        )
+    else:
+        logger.info("Constructing a test table")
+        ht_final = ht_original.annotate(
+            VRS_Allele=ht_annotated[ht_original.locus, ht_original.alleles].VRS_Allele
+        )
+
+    logger.info(f"Outputting final table at: {output_paths_dict[args.version]}")
+    ht_final.write(output_paths_dict[args.version],overwrite=args.overwrite)
+    # hl.write_table(ht_final,output_paths_dict[args.version])
+
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("--billing-project", help="Project to bill.", type=str)
+    parser.add_argument(
+        "--image",
+        default="us-central1-docker.pkg.dev/broad-mpg-gnomad/ga4gh-vrs/marten-vrs-image-v2-031323",
+        help="Image in a GCP Artifact Registry repository.",
+        type=str,
+    )
+    parser.add_argument(
+        "--working-bucket",
+        help="Name of GCP Bucket to mount for access.",
+        default="gnomad-vrs",
+        type=str,
+    )
+    parser.add_argument(
+        "--version", help="Version of HT to read in", default="test_v3_1k", type=str
+    )
+    parser.add_argument(
+        "--partitions-for-vcf-export",
+        help="Number of partitions to use when exporting the Table to a sharded VCF (each partition is exported to a separate VCF). This value determines the breakdown of jobs that will be ran (the VRS annotation script is ran in parallel on the VCFs).",
+        default=100,
+        type=int,
+    )
+    parser.add_argument(
+        "--prefix",
+        help="Prefix to append to names of all saved temp or output files.",
+        type=str,
+        default="",
+    )
+    parser.add_argument(
+        "--header-path",
+        help="Full path of txt file containing lines to append to VCF headers for fields that maybe be missing when exporting the Table to VCF.",
+        type=str,
+        default="gs://gnomad-vrs/header-fix.txt",
+    )
+    parser.add_argument(
+        "--downsample",
+        help="If reading in the whole Release Table, option to downsample",
+        default=1.00,
+        type=float,
+    )
+    parser.add_argument(
+        "--overwrite",
+        help="Boolean to pass to ht.write(overwrite=_____)",
+        action='store_true'
+    )
+
+    args = parser.parse_args()
+
+    main(args)
```

Comment:
i think you are supposed to avoid dashes in the names of scripts because it could cause import issues, also can drop "wrapper"

rename "vrs-annotation-batch-wrapper.py  -> vrs_annotation_batch.py"

Reply:
I don't think it will let me change it by adding a suggestion to a Batch of Suggestions, so expect a separate commit for this